magefile.go

```diff
@@ -1022,6 +1070,15 @@ func packageAgent(ctx context.Context, platforms []string, dependenciesVersion s
 	log.Println("--- Running post packaging ")
 	mg.Deps(Update)
 	mg.Deps(agentBinaryTarget, CrossBuildGoDaemon)
+
+	// compile the elastic-agent.exe proxy binary for the windows archive
+	for _, platform := range platforms {
+		if platform == "windows/amd64" {
+			mg.Deps(Build.WindowsArchiveRootBinary)
+			break
+		}
+	}
```

Comment:
```suggestion
	if slices.Contains(platforms, "windows/amd64") {
		mg.Deps(Build.WindowsArchiveRootBinary)
	}
```

Reply:
sure why not, let's utilise standard library 🙂 pushed a commit with it here fe889d436efbbcbaf43820b3a2711c9acf09919e please feel free to unresolve